_scss/_github.scss

```diff
@@ -24,13 +24,15 @@ github.com style (c) Vasily Polovnyov <vast@whiteants.net>
   font-weight: bold;
 }
 
+/*
```

Comment:
I found lot of commented code in this codebase. Maybe we should get rid of it

Reply:
Yes, I generally would clean up these. The reason I left them is that it looks like these were "overriding" the defaults in an "upstream" stylesheet. Keeping them commented makes it slightly easier to see what rules we removed/modified.

That said; I want to have another look at https://github.com/docker/docker.github.io/pull/10288 to remove highlightjs (and related styles) altogether, so that we only have a single location to style code-blocks / samples.